src/secret.rs

```diff
@@ -1,15 +1,12 @@
 //! Utilities for working with secret values. This module includes functionality for overwriting
 //! memory with zeros.
 
-use zeroize::Zeroize;
-
-use crate::{Fr, FrRepr};
+use crate::Fr;
+use group::ff::Field;
 
 /// Overwrites a single field element with zeros.
 pub(crate) fn clear_fr(fr: &mut Fr) {
-    // TODO: Remove this after pairing support `Zeroize`
-    let fr_repr = unsafe { &mut *(fr as *mut Fr as *mut FrRepr) };
-    fr_repr.0.zeroize();
+    *fr = Fr::zero();
```

Comment:
I don't think this is enough to securely erase the secret from memory. I *think* what this is doing is that it's changing a pointer to point to the new `Fr::zero()`, and then lets rust reclaim the old `Fr` secret eventually.

I don't think this is blocking though, lets log an issue and we can investigate a `Zeroize` impl in upstream blstrs